js/app/dashboard/components/LeafletPopupContent.js

```diff
@@ -26,7 +29,7 @@ export default class extends Component {
       <div className="pt-2">
         <header className="d-flex justify-content-between align-items-center mb-2">
           <strong>
-            { i18n.t('ADMIN_DASHBOARD_TASK_CAPTION_SHORT', { id: task.id }) }
+            { i18n.t('ADMIN_DASHBOARD_TASK_CAPTION_SHORT', { id: task.id }  ) } | { ( task.metadata.order_number && !_.isEmpty(task.metadata.order_number) )}
```

Comment:
This doesn't make any sense, the conditional returns a boolean, but it should be a string with the order number.